src/main/java/io/appform/dropwizard/actors/connectivity/RMQConnection.java

```diff
@@ -46,14 +46,17 @@
 public class RMQConnection implements Managed {
     @Getter
     private final RMQConfig config;
-    @VisibleForTesting
-    @Getter
+    private final String name;
     private Connection connection;
     private Channel channel;
     private final MetricRegistry metricRegistry;
     private final ExecutorService executorService;
 
-    public RMQConnection(RMQConfig config, MetricRegistry metricRegistry, ExecutorService executorService) {
+    public RMQConnection(String name,
+                         RMQConfig config,
+                         MetricRegistry metricRegistry,
+                         ExecutorService executorService) {
```

Comment:
Maybe pass env/lifecycle and register healthcheck here itself?

Reply:
Done